auto3dseg/algorithm_templates/swinunetr/scripts/algo.py

```diff
@@ -255,16 +293,14 @@ def objective(trial):
                 cmd += f"--num_images_per_batch {num_images_per_batch} "
                 cmd += f"--num_sw_batch_size {num_sw_batch_size} "
                 cmd += f"--validation_data_device {validation_data_device}"
-                _ = subprocess.run(cmd.split(), env=ps_environ, check=True)
-            except RuntimeError as e:
-                if "out of memory" in str(e):
-                    return (
-                        float(num_images_per_batch)
-                        * float(num_sw_batch_size)
-                        * device_factor
-                    )
-                else:
-                    raise(e)
+                _ = subprocess.run(cmd.split(), check=True)
+            except BaseException:
```

Comment:
Can we check if the exception has `out of memory` before printing `OOM`?

This is how it was processed before:
```
 except RuntimeError as e:
                if "out of memory" in str(e):
                    return (
                        float(num_images_per_batch)
                        * float(num_sw_batch_size)
                        * device_factor
                    )
                else:
                    raise(e)
```

Reply:
@dongyang0122

Reply:
Opened an issue here:
https://github.com/Project-MONAI/research-contributions/issues/222.